src/main/java/com/heatheryou/httpserver/CommandLineArgs.java

```diff
@@ -0,0 +1,18 @@
+package com.heatheryou.httpserver;
+
+public class CommandLineArgs {
+    public static boolean isValid(String[] args) {
```

Comment:
Is this supposed to be `static`?

`CommandLineArgs` is instantiated, and then this method is called like `commandLineArgs.isValid(args)`.